src/codec.spec.ts

```diff
@@ -132,7 +138,7 @@ describe("RLN codec with version 1", () => {
     const protoResult = await rlnDecoder.fromWireToProtoObj(bytes!);
 
     expect(protoResult).to.not.be.undefined;
-    const msg = (await rlnDecoder.fromProtoObj(protoResult!))!;
+    const msg = (await rlnDecoder.fromProtoObj("", protoResult!))!;
```

Comment:
nitpick: might be good to justify the use of an empty string. Maybe a glob const `DUMMY_CONTENT_TOPIC`.

Reply:
Good one, agree!